Installation/Action/hp.InviteWitnessAction.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$id": "https://hp.nl/action/hp.InviteWitnessAction.action.json",
+  "$schema": "https://docs.commongateway.nl/schemas/Action.schema.json",
+  "version": "0.0.4",
```

Comment:
Could be 0.0.1

Reply:
This is a action that was already in the repository. I only changed the location

Reply:
If the action is moved from installation.json it indeed should have a higher (or if the contents remain the same, the same) version number as in installation.json